R/min_grid.R

```diff
@@ -109,7 +109,7 @@ get_submodel_info <- function(spec) {
     dplyr::filter(engine == spec$engine) %>%
     dplyr::select(name = parsnip, has_submodel) %>%
     dplyr::full_join(
-      hardhat::extract_parameter_set_dials(spec) %>% tibble::as_tibble() %>% dplyr::select(name, id),
+      hardhat::extract_parameter_set_dials(spec) %>% dplyr::select(name, id),
```

Comment:
Note that `extract_parameter_set_dials.model_spec()` returns a tibble:

https://github.com/tidymodels/dials/blob/ec3cd5154bfae1b677c753861ea4330b11330bb9/R/parameters.R#L127-L137